README.md

```diff
@@ -95,21 +80,16 @@ A curated list of anime streaming/downloading websites.
 
 * [bato.to](http://bato.to/)
 * [bulumanga.com](http://bulumanga.com/)
-* [jaiminisbox.com](https://jaiminisbox.com/)
-* [mangafox.me](https://mangafox.me/)
+* [fanfox.net](https://fanfox.net/)
 * [mangafreak.eu](http://mangafreak.eu/)
-* [mangahere.co](https://www.mangahere.co/)
+* [mangahere.cc](https://mangahere.cc/)
 * [mangakakalot.com](http://mangakakalot.com/)
-* [manganel.com](http://manganel.com)
+* [manganelo.com](https://manganelo.com//)
 * [mangapanda.com](http://www.mangapanda.com/)
-* [mangapark.me](http://mangapark.me/)
+* [mangapark.net](http://mangapark.net/)
 * [mangareader.net](http://www.mangareader.net/)
-* [mangaseeonline.us](http://mangaseeonline.us/)
-* [mangastream.com](http://mangastream.com/)
 * [ReadManga Today](https://www.readmng.com/)
-* [zingbox.me](http://www.zingbox.me/)
 
 ### MANGA (mobile app)
 * [mangazone](https://play.google.com/store/apps/details?id=com.ng.mangazone&hl=en)
-* [Manga Rock](https://play.google.com/store/apps/details?id=com.notabasement.mangarock.android.titan)
```

Comment:
change to https://mangarock.com/getapp

Reply:
Mangarock is down so its useless.

Reply:
down? Site seems to be running, did they announce that it's down?

Reply:
yes ```Hi all

As we announced previously, we have removed all reading features from the website.

Now when you visit each title’s info page, you can no longer read scanlations. Instead, we have added a feature that links each title in Manga Rock to an official content provider’s site (like Comixology, or viz.com etc.) where you can support the original authors and their work.

Disclaimer:  We do not receive any referral fee from any of your purchases.

In the meantime, our team is still working hard on project “MR Comics” where we hope to make comics more accessible to everyone — we will share details with you soon.

If you have any questions or need help (membership, account etc.), please contact  support@mangarock.com```